Final Project Sprint 4 version 1

comrade-che opened the pull request.

Commit:
Final Project Sprint 4 version 1

mkouznetsov-git commented on pom.xml:
```diff
@@ -0,0 +1,55 @@
+
+    <dependencies>
+        <dependency>
+            <groupId>junit</groupId>
```
Дубликат.

mkouznetsov-git commented on pom.xml:
```diff
@@ -0,0 +1,55 @@
+            <version>4.19.1</version>
+        </dependency>
+        <dependency>
+            <groupId>io.github.bonigarcia</groupId>
```
Дубликат

mkouznetsov-git commented on src/test/java/ru/praktikum_services/qa_scooter/ScooterOrderTest.java:
```diff
@@ -0,0 +1,67 @@
+
+
+public class ScooterOrderTest {
+    private WebDriverFactory webdriverfactory = new WebDriverFactory();
```
webdriverfactory нигде не используется

Commit:
Final Project Sprint 4 version 2
Delete duplicate in pom.xml
Delete webdriverfactory in src/test/java/ru/praktikum_services/qa_scooter/ScooterOrderTest.java:

comrade-che commented:
@mkouznetsov-git Внесены изменения в version 2 согласно выданных замечаний
